src/orion/client/__init__.py

```diff
@@ -55,6 +55,7 @@ def build_experiment(
     heartbeat=None,
     working_dir=None,
     debug=False,
+    knowledge_base: "KnowledgeBase" = None,
```

Comment:
```suggestion
    knowledge_base = None,
```
Unless the rest in typed. :grin:

Reply:
Also, need to be added in the docstring.